README.md

```diff
@@ -510,6 +510,10 @@ services:
         # half a minute ("30s").
         backoff-limit: <duration>
 
+        # (Optional) The amount of time afforded to this service to handle
+        # SIGTERM and exit gracefully before SIGKILL terminates it forcefully.
+        # Default is 25 seconds ("25s").
```

Comment:
This feels like too much for a default. Imagine someone sitting on the terminal for 25 seconds while they run "pebble stop foo"... they will likely CTRL-C out before the job is done as it seems stuck anyway.

Reply:
This was changed to 25s to be inline with Kubernetes default 30s. Allowing enough time by default for processes that require a little extra time. My understanding was this was an agreed upon item.

However happy to change. Should we take this back to 5s or another value? @benhoyt do you have any thoughts on what this value should be?

Reply:
I think the point is that in cases where there is more direct user interaction, 25s feels too long, where 5s doesn't.

Presumably this can still be set to 25/30s when Juju writes the plan for the unit agent either way, and still solve our original issue?

Reply:
Yeah, agreed re a user interacting with `pebble stop`. I reckon let's keep the default 5s and Juju can easily override.

Reply:
Easy done.